src/liballoc/collections/vec_deque.rs

```diff
@@ -2709,6 +2709,33 @@ impl<T: fmt::Debug> fmt::Debug for VecDeque<T> {
 
 #[stable(feature = "vecdeque_vec_conversions", since = "1.10.0")]
 impl<T> From<Vec<T>> for VecDeque<T> {
+    /// Turn a `Vec<T>` into a `VecDeque<T>`.
+    ///
+    /// This avoids reallocating where possible, but the conditions for that are
+    /// strict, and subject to change, and so shouldn't be relied upon unless the
+    /// `Vec<T>` came from `From<VecDeque<T>>` has hasn't been reallocated.
```

Comment:
```suggestion
    /// `Vec<T>` came from `From<VecDeque<T>>` and hasn't been reallocated.
```

(apologies for the former comment, I had trouble parsing this because of the duplicated word, and figured it out after reading the example)